meta-ostro/classes/ostro-image.bbclass

```diff
@@ -156,7 +280,21 @@ FEATURE_PACKAGES_soletta-tools = "soletta-dev-app"
 # because it is the most common source code management tool.
 FEATURE_PACKAGES_tools-develop = "packagegroup-core-buildessential git"
 
-FEATURE_PACKAGES_swupd = "packagegroup-swupd"
+# Add bash because it is more convenient to use than dash.
+# This does not change /bin/sh due to re-organized update-alternative
+# priorities.
+FEATURE_PACKAGES_tools-interactive = "packagegroup-tools-interactive bash"
+
+# We could make bash the login shell for interactive accounts as shown
+# below, but that would have to be done also in the os-core and thus
+# tools-interactive would have to be set in all swupd images.
+# TODO (?): introduce a bash-login-shell image feature?
+# ROOTFS_POSTPROCESS_COMMAND_append = "${@bb.utils.contains('IMAGE_FEATURES', 'tools-interactive', ' root_bash_shell; ', '', d)}"
+# root_bash_shell () {
+#     sed -i -e 's;/bin/sh;/bin/bash;' \
+#        ${IMAGE_ROOTFS}${sysconfdir}/passwd \
+#        ${IMAGE_ROOTFS}${sysconfdir}/default/useradd
```

Comment:
when we move interactive stuff into a container with its own stuff, we can define the root login inside the container to use /bin/bash. Actually bash can be provided only in that container.